README.md

```diff
@@ -1,8 +1,10 @@
 # [opensourcedesign.net](https://opensourcedesign.net)
 
+[![GitHub issues](https://img.shields.io/github/issues/opensourcedesign/opensourcedesign.github.io?logo=github)](https://github.com/opensourcedesign/opensourcedesign.github.io/issues) [![GitHub forks](https://img.shields.io/github/forks/opensourcedesign/opensourcedesign.github.io?logo=github)](https://github.com/opensourcedesign/opensourcedesign.github.io/network) [![GitHub stars](https://img.shields.io/github/stars/opensourcedesign/opensourcedesign.github.io?logo=github)](https://github.com/opensourcedesign/opensourcedesign.github.io/stargazers) [![GitHub license](https://img.shields.io/github/license/opensourcedesign/opensourcedesign.github.io?logo=apache)](https://github.com/opensourcedesign/opensourcedesign.github.io/blob/master/LICENSE)
```

Comment:
I feel like all of this info duplicates the GitHub interface? What’s the reasoning of adding it to the readme, when all of this info shows already at the top of the repository? :)